tdf_simulator/config.py

```diff
@@ -148,8 +189,12 @@ def from_dict(cls, d: dict[str, Any]) -> WindowInfo:
             collision_energy=d["CollisionEnergy"],
         )
 
+    def to_dict(self) -> dict:
+        return asdict(self)
```

Comment:
This does not work properly as names as incorrect. Quick fix is to replace with:
```python
        return {
            "WindowGroup": self.window_group,
            "ScanNumBegin": self.scan_num_begin,
            "ScanNumEnd": self.scan_num_end,
            "IsolationMz": self.isolation_mz,
            "IsolationWidth": self.isolation_width,
            "CollisionEnergy": self.collision_energy,
        }
```